go.mod

```diff
@@ -70,6 +70,7 @@ require (
 	golang.org/x/tools v0.0.0-20200626171337-aa94e735be7f // indirect
 	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f // indirect
 	gopkg.in/ini.v1 v1.57.0 // indirect
+	gopkg.in/src-d/go-log.v1 v1.0.1
```

Comment:
please rebase on master and make sure this import is not here.

Reply:
rebasing won't help since the CS is in the branch and will be reapplied. `git checkout master -- go.mod`